Address bug in case sensitivity

smithjac opened the pull request:
Bug fixed where only the dockets collection was being queried without case sensitivity. All collections are now queried without case sensitivity.

adams-k commented:
Some of the files changed here are outdated and would override the current versions, an example being the webapp-testing.yml file. These will still need to get fixed for this can be merged.

Commit:
Address bug in case sensitivity

Bug fixed where only the dockets collection was being
queried without case sensitivity. All collections
are now queried without case sensitivity.

conwaycolin commented:
I pulled this and it all looks good. The bug is fixed I'm getting the correct results.
